### ISSUE TITLE
Some TensorFlow implementation problems.

### ISSUE THREAD

**Issue by zengpz** [bermanmaxim/LovaszSoftmax#7]
hi, Maxim

you present a very interesting and solid work! but I met some implementation error while using your lovasz loss in my deeplab v3+. My initial loss is tf.losses.softmax_cross_entropy, and I prepared: 

onehot_labels: `[batch_size, num_classes]` target one-hot-encoded labels.
logits: `[batch_size, num_classes]` logits outputs of the network .

as its input, but it just didn't fit in your loss, could you please give some advice about how to transfer original params into your params like probas, labels? Thank you!

**Comment by bermanmaxim** [comment]
Hello,
Indeed, the loss does not expect a one-hot label encoding, but simply the integer labels that are expected. See [this line](https://github.com/bermanmaxim/LovaszSoftmax/blob/bd7e017173692b9824423e3b288b651ed526fc50/tensorflow/lovasz_losses_tf.py#L99) for the expected inputs.
You may change your data loader to provide integer labelled images, or convert one-hot to integer with e.g. argmax, altough that would be some waste of computation.
